api_coverage.md

```diff
@@ -0,0 +1,108 @@
+<!--
+  -- Copyright (c) 2020, Arm Limited, All Rights Reserved
+  -- SPDX-License-Identifier: Apache-2.0
+  --
+  -- Licensed under the Apache License, Version 2.0 (the "License"); you may
+  -- not use this file except in compliance with the License.
+  -- You may obtain a copy of the License at
+  --
+  -- http://www.apache.org/licenses/LICENSE-2.0
+  --
+  -- Unless required by applicable law or agreed to in writing, software
+  -- distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  -- WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  -- See the License for the specific language governing permissions and
+  -- limitations under the License.
+--->
+
+# PSA Cryptography API coverage within Parsec
+
+This page describes the current state of Parsec support for the operations, attributes and options that are defined in the PSA Cryptography API 1.0.0. It covers two different sides of the issue:
+
+* Coverage of the suite of operations that have been [defined for IPC](https://github.com/parallaxsecond/parsec-operations) and which can be requested from the service.
+* Coverage of the operations and key attributes implemented for each provider (that is offered as part of the Parsec service).
+
+**NOTE:** This does not guarantee or imply the same amount of support within client libraries. We encourage each library to publish its own coverage figures in their format of choice.
+
+## IPC operations coverage
+
+The table below shows coverage only for single part operations. Multi-part operations will be added in the future and will be organized by operation type.
+
+### Single part operations
+
+| Operation name                   | Protobuf contract exists                                                                                               |
+| :------------------------------- | :--------------------------------------------------------------------------------------------------------------------: |
+| psa_import_key                   | [:white_check_mark:](https://github.com/parallaxsecond/parsec-operations/blob/master/protobuf/import_key.proto)        |
```

Comment:
As those are functions, it would maybe be nice to put backticks around: `psa_import_key`